python/pyspark/taskcontext.py

```diff
@@ -144,10 +144,19 @@ def __init__(self):
         """Construct a BarrierTaskContext, use get instead"""
         pass
 
+    def __new__(cls):
```

Comment:
Why should we rewrite `__new__`? Can't we do this in `_getOrCreate` as well?

Reply:
Yep, do this in `_getOrCreate` has same effect, this is an over consider of https://github.com/apache/spark/blob/aec0af4a952df2957e21d39d1e0546a36ab7ab86/python/pyspark/taskcontext.py#L44-L45
Deleted in 02555b8.